core/row/row.pl.sdoc

```diff
@@ -97,7 +97,23 @@ defoperator row_sort => q{
     '--buffer-size='      . conf 'row/sort-buffer',
     '--parallel='         . conf 'row/sort-parallel'), @_};
 
-defshort '/g', pmap q{row_sort_op        sort_args @$_}, sortspec;
+defoperator partial_sort => q{
+  my $sort_size = shift;
+  my @buff = ();
+  while (<STDIN>) {
+    push @buff, $_;
+    if (@buff == $sort_size) {
+      print join "", sort(@buff);
```

Comment:
ok, i got rid of that; is this operator a good idea? I wrote it thinking mainly of hadoop workflows where you have tons of near-dupes to unique and sometimes `ggABu` will make you have to start spilling to disk and murdering performance.